app/helpers/codelist_helper.rb

```diff
@@ -34,6 +34,11 @@ def region_select_options
     objects.unshift(OpenStruct.new(name: "Developing countries, unspecified", code: "998")).uniq
   end
 
+  def country_select_options
+    objects = yaml_to_objects(entity: "activity", type: "recipient_country", with_empty_item: false)
+    objects.unshift(OpenStruct.new(name: t("page_content.activity.recipient_country.default_selection_value"), code: "")).uniq
```

Comment:
What was the motivation for using i18n for this value, when the values near it are inline?

Reply:
The motivation was probably me asking for it to be internationalised 😆 

I hadn't realised the others weren't internationalised. Internationalise all the things!!

Reply:
@lozette  review: 

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/249#discussion_r389724181

Reply:
Ah ok. I think it feels a little weird because some default values in here are real values, while others are placeholders. It's hard to tell from the implementation which is which. I'd love for that to be clarified better. Not a thing for now, though, I suspect.